test/db/analysis/arm

```diff
@@ -1067,13 +1067,13 @@ RUN
 NAME=misaligned arm string xref
 FILE=bins/mach0/misaligned_data-iOS-armv7
 CMDS=<<EOF
-aav
-aae
-af
-axt @ str.helloradareworld
+aaa
+axt @ str.helloradareworl
```

Comment:
Any clue why it now cuts more than before?

Reply:
no idea, i think is the filter doing stuff.